### PR TITLE
fix compatibility with flask 2.3.2

### PR DESCRIPTION
**Pull request by allusernamestakenexceptthis**
Fix for a compatibility issue in app.py

### CODE REVIEW

**Comment by sasanquaneuf** [comment]
@allusernamestakenexceptthis 
Thanks, could you tell me the result of `pip3 freeze`?
(I think that the version of markupsafe isn't suitable for flask 2.3.2. I'll fix other issues with the versions of dependencies.)

**Comment by allusernamestakenexceptthis** [comment]
> @allusernamestakenexceptthis Thanks, could you tell me the result of `pip3 freeze`? (I think that the version of markupsafe isn't suitable for flask 2.3.2. I'll fix other issues with the versions of dependencies.)

Thanks for taking a look.

Here is the output of pip3.8 freeze:

```
alembic==1.11.1
autocommand==2.2.2
Babel==2.12.1
blinker==1.6.2
boto3==1.26.137
botocore==1.29.137
cffi==1.15.1
cheroot==9.0.0
CherryPy==18.8.0
click==8.1.3
cryptography==40.0.2
exceptiongroup==1.1.1
Flask==2.3.2
flask-babel==3.1.0
Flask-WTF==1.1.1
greenlet==2.0.2
importlib-metadata==6.6.0
importlib-resources==5.12.0
inflect==6.0.4
iniconfig==2.0.0
itsdangerous==2.1.2
jaraco.collections==4.1.0
jaraco.context==4.3.0
jaraco.functools==3.6.0
jaraco.text==3.11.1
Jinja2==3.1.2
jmespath==1.0.1
Mako==1.2.4
MarkupSafe==2.1.1
more-itertools==9.1.0
mysql-connector==2.2.9
packaging==23.1
Paste==3.5.3
pluggy==1.0.0
portend==3.1.0
pycparser==2.21
pydantic==1.10.7
PyJWT==2.7.0
pytest==7.3.1
python-dateutil==2.8.2
pytz==2023.3
s3transfer==0.6.1
six==1.16.0
SQLAlchemy==2.0.15
tempora==5.2.2
tomli==2.0.1
typing-extensions==4.5.0
urllib3==1.26.15
Werkzeug==2.3.4
WTForms==3.0.1
zc.lockfile==3.0.post1
zipp==3.15.0
``` 

I installed markupsafe 2.1.1 because when I installed the requirements it complained that Werzeug and Jinja2 required higher version.  After that, I haven't particularly tested if markupsafe was working properly as intended. Though there wasn't any related error.

**Comment by sasanquaneuf** [comment]
@allusernamestakenexceptthis 
Thank you so match!

> I installed markupsafe 2.1.1 because when I installed the requirements it complained that Werzeug and Jinja2 required higher version.

Okay, I got it. I'll merge this PR and later add a github action to check if dependencies are broken.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# pullre-kun
+# pullre-kun (Forked)
```
Comment (sasanquaneuf) [inline README.md]: Can you remove "(Forked)"?

Comment (allusernamestakenexceptthis) [reply README.md]: Ok, I reverted Readme.md and pushed the modification.  

Sorry for adding that. I was testing pull requests with the app. but I should've done it with different repo.

**Review by sasanquaneuf — APPROVED** [review]
LGTM

**Pull request merged**